src/super_gradients/training/models/__init__.py

```diff
@@ -163,8 +164,18 @@ def inner(*args, **kwargs):
     "[-] from super_gradients.training.models import BasicBlock\n"
     "[+] from super_gradients.training.models import BasicResNetBlock\n",
 )
+HpmStruct = make_deprecated(
+    func=CurrVersionHpmStruct,
+    reason="You're importing `HpmStruct` class from `super_gradients.training.models`. This is deprecated since SuperGradients 3.1.0.\n"
+    "Please update your code to import it as follows:\n"
+    "[-] from super_gradients.training.models import HpmStruct\n"
+    "[+] from super_gradients.training.utils import HpmStruct\n",
+)
+
 
 __all__ = [
+    "HpmStruct",
+    "Bottleneck",
```

Comment:
where is the deprecation for this?

Reply:
slipped out, fixed in 1ae6315af6e40de9205e517096c7200986e57ec5